src/components/ContactDetails/ContactDetailsProperty.vue

```diff
@@ -103,6 +112,20 @@ export default {
 			type: Array,
 			default: () => [],
 		},
+		bus: {
+			type: Object,
+			required: true,
+		},
+	},
+
+	created() {
+		this.bus.$on('focusProp', async (id) => {
+			if (id === this.propName) {
+				this.focus = true
```

Comment:
would it be possible to use native focus with `this.refs.component.focus()` or similar? The styling can be done with the pseudo class `:focus`, if not existent already

Reply:
I focus the first input in the prop now :+1: